src/lambda/template.yaml

```diff
@@ -41,19 +41,32 @@ Resources:
             - glue:StartWorkflowRun
             - glue:PutWorkflowRunProperties
             Resource:
-            - !Sub arn:aws:glue:${AWS::Region}:${AWS::AccountId}:workflow/${WorkflowName}
+            - !Sub arn:aws:glue:${AWS::Region}:${AWS::AccountId}:workflow/*
```

Comment:
Why the switch to all workflows here?

Reply:
It used to be the case that we had one lambda per study. We now have one lambda per app. So a lambda will poll an SQS queue which is subscribed to the app-wide Bridge SNS topic, receiving messages for data from all different types of studies. The lambda then groups these messages by study and sends them to the appropriate study-specific workflow.